### PR TITLE
lang/python/python-mako: Add at 1.2.4

### PR DESCRIPTION
**Pull request by joebonrichie**
Initially included for the mesa build

### CODE REVIEW

**Comment by ikeycode** [comment]
I like - and it highlights the need to get automatic python dependencies working in boulder.
(Obviously just need that `monitoring.yml` and I'll merge)

File: lang/python/python-mako/monitoring.yml
Comment (ReillyBrogan) [inline lang/python/python-mako/monitoring.yml]: I think the actual ID is `3915`. ([ref](https://release-monitoring.org/project/3915/))

97183 seems to be [this](https://release-monitoring.org/project/97183/)

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

**Comment by ReillyBrogan** [comment]
LGTM, I'm going to add the CPE after merging it

**Pull request merged**
